### DISCUSSION TITLE
Cannot find Flow Deck V2 during I2C scan on Raspberry Pi

### DISCUSSION THREAD

**Discussion by Andrew123098** [bitcraze/discussions#1286] [Q&A] [answered]
Hello!
I am working on a drone (not the crazyflie) and wanting to use the Flow Deck V2. I am currently trying to test the Flow Deck with a Rasberry Pi Zero 2 w to see if I can get usable data out of it. According to [this post](https://github.com/bitcraze/Bitcraze_PMW3901/issues/3), I should be able to. However, the raspberry pi does not seem able to find the VL53L1X distance sensor when scanning for I2C connections using < i2cdetect -y 1 >. I am powering the Flow Deck with 3.300 volts and I am certain it is properly wired up (I have directly connected VCC-->3v3, GND-->GND, SDA-->SDA and SCL-->SCL) directly from the flow deck to the raspberry pi with no breadboard or other electrical components. I have tested the connections with a multimeter to make sure they are correct and tested the supply voltage. I have changed the baud rate in the boot config file to 400 kHz which according to [my research](https://community.st.com/t5/mems-sensors/maximum-i2c-frequency-for-vl53l1x/td-p/183175) is within the limit of the VL53L1X. I have enabled the I2C communications in the raspi-config as well (and successfully tested another I2C sensor at a different address). Is there something I am missing? Do I need to include some other electrical components to make this work? Is it even possible to use the Flow Deck V2 with a general MCU/Microcontroller that isn't the crazyflie like Arduino or Raspberry Pi?
Thank you so much in advance for your help. Any advice would be appreciated and please let me know if there is any other information I can provide.
Andrew

**Comment by knmcguire** [comment]
Hi! I see you have closed the issue. Could you leave it open, or else it dissappears from the discussion list.

Did you find a solution for your problem? Could you update us on the fix?

**Reply by Andrew123098** [reply]
Hi yes, fortunately (or unfortunately depending on how you look at it) the problem fixed itself. I changed nothing in the setup and the problem simply went away the next day. So with the setup described above I was able to to get data from the distance sensor on my raspberry pi.

**Comment by Andrew123098** [comment, marked as answer]
SOLVED: There apparently was no issue with my setup. I tried again with the same setup the next day and I was able to stream data from the Flow Deck V2 to my raspberry pi. I do not know what the problem was before. Perhaps something wrong with my wiring? But if you follow what I describe above you can get data from the flow deck v2 distance sensor on the raspberry pi.

**Reply by knmcguire** [reply]
thanks for the update!
